.github/workflows/build.yml

```diff
@@ -0,0 +1,91 @@
+name: Docker build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  schedule:
+    - cron: "0 1 * * *" # Daily “At 01:00” UTC
+  workflow_dispatch:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        cuda: ["11.8.0"]
+        python: ["3.10", "3.11", "3.12"]
+        linux: ["ubuntu20.04"]
+        rapids: ["24.10", "24.12"]
+        image:
+          - tag: "gpuci/dask"
+            context: "./dask"
+          - tag: "gpuci/distributed"
+            context: "./distributed"
+          - tag: "gpuci/dask_image"
+            context: "./dask_image"
+        # dask-image gpuCI isn't dependent on RAPIDS
+        exclude:
+          - image: "gpuci/dask_image"
+            rapids: "24.10"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        if: github.repository == 'rapidsai/dask-build-environment' && (github.event_name == 'push' || github.event_name == 'workflow_dispatch')
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
These are just filler for now, assuming these don't exist - @ajschmidt8 are there org/repo secrets we could use here to authenticate with the `gpuci` dockerhub?

https://hub.docker.com/u/gpuci

Reply:
I would check with @raydouglass. I think that DockerHub organization is largely defunct except for the Dask/Distributed images.

Ray will have guidance on whether you should keep using it. And if so, he can also provide credentials.